devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -218,20 +219,62 @@ def ping() {
 }
 
 def refresh() {
+    log.debug "Refresh..."
 	delayBetween([
 		zwave.switchBinaryV1.switchBinaryGet().format(),
-		zwave.meterV2.meterGet(scale: 0).format(),
-		zwave.meterV2.meterGet(scale: 2).format()
+		meterGet(scale: 0).format(),
+		meterGet(scale: 2).format()
 	])
 }
 
 def configure() {
```

Comment:
You'll want to make sure you put some sort of call to get the manufacturer info (if it is not already present) in the `installed()` method so that you can be more confident that it exists by the time this method is called.

Reply:
@greens Can you provide more clarification on this. Thanks.